app/src/renderer/components/wallet/PageWallet.vue

```diff
@@ -6,7 +6,7 @@
         <a
           v-tooltip.bottom="'Refresh'"
           :disabled="!connected"
-          @click="connected && updateBalances()"
+          @click="connected && queryWalletBalances()"
```

Comment:
ok thanks. not a big deal - but seems like it would be better for this to be hidden or a loading graphic instead of being present and clickable but if `!connected` then nothing happens.

Reply:
I think our design of disabled buttons should be self explaining. Currently they are not styled differently. From a UX perspective hiding actionables is a bad idea I thought.

Reply:
ah you are so right! it will be disabled. great. so this is just _extra_ disabled 😉